recipes/khmer/fe0ce11645/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "fe0ce116456b296c522ba24294a0cabce3b2648b" %}
+{% set sha256 = "b6da62bb477414a59fa1283e6c00a4ecf96728b1ce7981ec2aaa1058987974d0" %}
+
+package:
+  name: khmer
+  version: '{{version}}'
```

Comment:
I don't think this is a good idea. Its not clear which version conda will favor and is the "latest". Please ask for a release instead ...

Reply:
Call it 3.0.0a2 if you must ... but that won't change that everyone will get it on their desk.

The thing is that Bioconda has no mechanism for distributing previews at this time.